save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ProjectController.kt

```diff
@@ -149,6 +155,9 @@ class ProjectController(private val projectService: ProjectService,
             val saveGit = gitService.saveGit(it, projectId)
             log.info("Save new git id = ${saveGit.id}")
         }
+        val project = projectService.findById(projectId).get()
+        val owner = userRepository.findById(userId).get()
+        lnkUserProjectService.setRole(owner, project, Role.OWNER)
```

Comment:
That's a shame that we need two DB queries only to comply with JPA model, while in reality mapping uses only IDs that we already have. However I'm not sure how it can be changed and still be readable. Maybe using `@Query` in repository class and creating a new method, that accepts only IDs

Reply:
Yes, something like this.

Reply:
Done